packages/layouts-css/src/page.scss

```diff
@@ -46,22 +46,97 @@
       align-items: center;
 
       //#region Add env(safe-area-inset-*) for iPhone notch on the left & right sides of content area
+
       padding: var(--iui-size-xl) unquote('max(var(--iui-size-2xl), env(safe-area-inset-right))') var(--iui-size-xl) unquote('max(var(--iui-size-2xl), env(safe-area-inset-left))');
 
       @include small-monitor {
-        padding: var(--iui-size-l) unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-right))') var(--iui-size-l) unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-left))');
+        padding: var(--iui-size-l)
+          unquote(
+            'max(calc(var(--iui-size-l) * 2), env(safe-area-inset-right))'
+          )
+          var(--iui-size-l)
+          unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-left))');
       }
 
       @include tablet {
-        padding: var(--iui-size-m) unquote('max(var(--iui-size-xl), env(safe-area-inset-right))') var(--iui-size-m) unquote('max(var(--iui-size-xl), env(safe-area-inset-left))');
+        padding: var(--iui-size-m)
+          unquote('max(var(--iui-size-xl), env(safe-area-inset-right))')
+          var(--iui-size-m)
+          unquote('max(var(--iui-size-xl), env(safe-area-inset-left))');
       }
 
       @include mobile {
-        padding: var(--iui-size-xs) unquote('max(var(--iui-size-l), env(safe-area-inset-right))') var(--iui-size-xs) unquote('max(var(--iui-size-l), env(safe-area-inset-left))');
+        padding: var(--iui-size-xs)
+          unquote('max(var(--iui-size-l), env(safe-area-inset-right))')
+          var(--iui-size-xs)
+          unquote('max(var(--iui-size-l), env(safe-area-inset-left))');
       }
       //#endregion Add env(safe-area-inset-*) for iPhone notch on the left & right sides of content area
     }
 
+
+    &-tools {
```

Comment:
Why did you add tools here?

Reply:
It was there before. I think it is for the styling of `ToolsArea.tsx` component that is used in layout demo 2.